crates/stellar-scaffold-cli/src/commands/build/clients.rs

```diff
@@ -651,6 +651,18 @@ export default new Client.Client({{
         Ok(())
     }
 
+    fn get_package_dir(&self, name: &str) -> Result<std::path::PathBuf, Error> {
```

Comment:
This works and if you merge as-is I won't complain but an `Option` might make more sense. It's not really an _error_ if there's no package directory, it just either exists or it doesn't.

Reply:
ahhh, yeah, you're correct there. I may go ahead and merge as-is for the bug fix but that Option is nice.